app/src/main/java/org/sopt/and/WavveApplication.kt

```diff
@@ -0,0 +1,11 @@
+package org.sopt.and
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+
+@HiltAndroidApp
+class WavveApplication : Application() {
+    override fun onCreate() {
+        super.onCreate()
+    }
+}
```

Comment:
```suggestion
class WavveApplication : Application()
```

이렇게 줄일 수 있습니다. 필요한 경우에 따로 onCreate 선언해보세요!

Reply:
오 onCreate를 선언하지 않아도 괜찮군요!!